docs/user_manual/components.md

```diff
@@ -181,6 +202,28 @@ It can happen that `tap_min > tap_max`. In this case the winding voltage is decr
 increased.
 ```
 
+#### Electric Model
+`transformer` is described by a π model, where $z_{series}$ and $y_{shunt}$ can be computed by following equations:
+
+$$
+   \begin{eqnarray} 
+        z_{base} = sn/(u2^2),
```

Comment:
It would be nice to highlight properties actually in use. 
(Also remove ones which are not wherever possible) The details about base are for our understanding and user wont have to concern with. For eg. `base_power_3p` can definitely be removed, while other bases can be kept if they make understanding better.